projects/effortless.md

```diff
@@ -0,0 +1,41 @@
+# Effortless
+
+The Effortless project is ...?
+
+**Slack Channel** #effortless
+
+**Triage Schedule**: TBD
+
+### Project Members
+
+#### Project Owners
+
+John Snow
+  - Github: [thelunaticscripter](https://github.com/thelunaticscripter)
+  - Slack: @thelunaticscripter
+
+Josh O'Brien
+  - Github: [OBrienCommaJosh](https://github.com/OBrienCommaJosh)
+  - Slack: @job
+
+Mark Dennard
+  - Github: [mdennard](https://github.com/mdennard)
+
+#### Project Approvers
+
```

Comment:
I think we should also add Jon Cowie to reviewers.

Reply:
Added.

Reply:
Added.